clean code style.

BillZhou734 opened the pull request:
1. modify 'gw' prefix from file/func name.
2. code style clean.

using "checkpatches.sh" pass.

jiaweiwsz commented:
Hello Bill and Roni,

Do we need use doca_flow* as prefix in the each API function or only doca*?

Thanks.

BillZhou734 commented:
for doca_dpdk.c , should use "doca_dpdk_x" as prefix
for doca_flow.c , should use "doca_flow_x" as prefix.

that means:  "doca_pipeline_add_entry"  -> "doca_flow_pipeline_add_entry"   ?

jiaweiwsz commented:
At least for exposed API to customer,  should be 'doca_flow*". Similar as doca_dpi* now in Doca Gerrit code.

BillZhou734 committed:
mv doca_gw_dpdk.c to doca_dpdk.c
mv doca_gw.c to doca_flow.c

Signed-off-by: Dong Zhou <dongzhou@nvidia.com>

BillZhou734 committed:
mv doca_gw_port to doca_flow_port
mv doca_gw_port_type to doca_flow_port_type
mv doca_gw_match to doca_flow_match
mv doca_gw_actions to doca_flow_actions
mv doca_gw_monitor to doca_flow_monitor
mv doca_gw_error to doca_flow_error
mv doca_gw_cfg to doca_flow_cfg

Signed-off-by: Dong Zhou <dongzhou@nvidia.com>

BillZhou734 committed:
mv doca_gw_pipeline to doca_flow_pipeline
mv doca_gw_pipeline_cfg to doca_flow_pipeline_cfg
mv doca_gw_pipelne_entry to doca_flow_pipeline_entry

Signed-off-by: Dong Zhou <dongzhou@nvidia.com>

BillZhou734 committed:
mv doca_gw_pipe_dpdk_flow to doca_dpdk_pipeline

Signed-off-by: Dong Zhou <dongzhou@nvidia.com>

BillZhou734 committed:
clean doca_dpdk.c file

Signed-off-by: Dong Zhou <dongzhou@nvidia.com>

BillZhou734 committed:
clean doca_flow.c and doca_flow.h

Signed-off-by: Dong Zhou <dongzhou@nvidia.com>

BillZhou734 committed:
clean doca_debug_dpdk.c and doca_debug_dpdk.h

Signed-off-by: Dong Zhou <dongzhou@nvidia.com>

BillZhou734 committed:
mv doca_fwd_table_cfg to doca_flow_fwd_table_cfg
mv doca_fwd_tbl to doca_flow_fwd_tbl

Signed-off-by: Dong Zhou <dongzhou@nvidia.com>

BillZhou734 committed:
move meter RSS action to pipeline creataion.

Signed-off-by: Dong Zhou <dongzhou@nvidia.com>
